test/accumulate-test.js

```diff
@@ -1,68 +1,68 @@
-require('buster').spec.expose();
-var expect = require('buster').expect;
-
-var Stream = require('../lib/Stream');
-var accumulate = require('../lib/combinator/accumulate');
-var throwError = require('../lib/combinator/errors').throwError;
-var observe = require('../lib/combinator/observe').observe;
-var drain = require('../lib/combinator/observe').drain;
-var fromArray = require('../lib/source/fromArray').fromArray;
-var create = require('../lib/source/create').create;
-var core = require('../lib/source/core');
-
-var scan = accumulate.scan;
-var reduce = accumulate.reduce;
-
-var empty = core.empty;
-var streamOf = core.of;
-
-var FakeDisposeSource = require('./helper/FakeDisposeSource');
-
-var sentinel = { value: 'sentinel' };
-var other = { value: 'other' };
+import { spec, referee } from 'buster';
+const { describe, it } = spec
+const { fail, assert } = referee;
+
+import Stream from '../lib/Stream'
+import { scan, reduce } from '../lib/combinator/accumulate'
+import { throwError } from '../lib/combinator/errors'
+import { observe, drain } from '../lib/combinator/observe'
+
+import { fromArray } from '../lib/source/fromArray'
+import { empty, of as just } from '../lib/source/core'
+
+import FakeDisposeSource from './helper/FakeDisposeSource'
+
+const sentinel = { value: 'sentinel' };
+const other = { value: 'other' };
+
+function endWith (endValue, { source }) {
```

Comment:
This could be useful in other tests.  I'm ok with being lazy about extracting it, iow waiting until there's a need :)